website/frontend/src/components/CleanAir/Hightlights/MainHighlight.js

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { getAllEvents } from '../../../../reduxStore/Events/EventSlice';
+import { getAllEvents } from '../../../../reduxStore/Events';
```

Comment:
Hi @OchiengPaul442 , I hope we can have simple import statements, these lenghty backward paths can be prone to human errors. :) I remember we had some sort of library to support simpler paths?

Reply:
thanks @Baalmart,
we do have the library installed, though I might have forgotten and used the relative paths in this case.